importer/main.py

```diff
@@ -861,8 +862,75 @@ def clean_duplicates(users, cascade_delete):
                 logging.info("No Practitioners found")
 
 
+# Create a csv file and initialize the CSV writer
+def write_csv(data, resource_type, fieldnames):
+    logging.info("Writing to csv file")
+    current_time = datetime.now().strftime("%Y-%m-%d-%H-%M")
+    csv_file = f"csv/{current_time}-export_{resource_type}.csv"
+    with open(csv_file, 'w', newline='') as file:
+        csv_writer = csv.writer(file)
+        csv_writer.writerow(fieldnames)
+        csv_writer.writerows(data)
+
+
+# This function exports resources from the API to a csv file
+def export_resources_to_csv(resource_type, parameter, value, batch_size):
+    resource_type = get_valid_resource_type(resource_type)
```

Comment:
let's change this (include in the doc ) to require the resource type exactly as it would be/work in a url so we don't have to do this. Might be a it confusing